src/test/java/patching/TestPatchManager.java

```diff
@@ -114,9 +116,83 @@ public void testNotificationHandler() throws IOException, ClassNotFoundException
         } catch (InterruptedException e) {
             e.printStackTrace();
         }
-
     }
 
+//    @Test
```

Comment:
It's a test-case in progress, that we will need in future. I was trying to figure out how to test what i was looking for, and this didn't quite hit the edge case.